src/qobj/boolean_functions.jl

```diff
@@ -6,71 +6,79 @@ export isket, isbra, isoper, isoperbra, isoperket, issuper
 export isunitary
```

Comment:
Should we also add a new function `isconstant` in the end of first line ?
I saw that QuTiP also got this function